account_move_transactionid_import/models/account_journal.py

```diff
@@ -12,3 +12,9 @@ class AccountJournal(models.Model):
             ('generic_csvxls_transaction',
              'Generic .csv/.xls based on SO transaction ID')
         ])
+
+    @api.onchange('import_type')
+    def onchange_import_type(self):
+        super(AccountJournal, self).onchange_import_type()
+        if self.import_type == 'generic_csvxls_transaction':
+            self.create_counterpart = True
```

Comment:
Again, it should really be `True` for create_counterpart and `False` for split_counterpart by default, as most imports worked that way in 7.0/8.0.